assets/chat/css/style.scss

```diff
@@ -1491,6 +1502,23 @@ hr {
       &.creator {
         color: $text-color1;
       }
+      &.favorite {
+        cursor: pointer;
+        margin-top: 5px;
+        .btn-icon {
+          transition: background-color 500ms;
+          width: 20px;
+          height: 20px;
```

Comment:
With a square element it's better to specify the size in a single dimension and an aspect ratio of 1.